.github/workflows/conformance-runtime.yaml

```
@@ -440,7 +441,7 @@ jobs:
           junit-directory: "cilium-junits"
 
   commit-status-final:
-    if: ${{ always() }}
+    if: ${{ github.event_name != 'push' }}
```

```suggestion
    if: ${{ always() && github.event_name != 'push' }}
```

Passing by comment, but I guess `always()` should still be present, otherwise it is skipped on failure.

---

Thanks for passing by. I thought that this is only necessary when leaving the default. But i double-checked on a test project - and it seems you're right :confused: 

Anyway, thanks a lot for passing by!

PS: I fixed it for the ones that i changed in #29404 too